src/pages/DashboardPage.vue

```diff
@@ -129,20 +119,24 @@ export default class DashboardPage extends Vue {
   }
 
   changeDashboardId(): void {
```

Comment:
This starts a dialog that eventually calls another function named changeDashboardId. The same function name for these 2 different but related functions is a bit confusing.

Reply:
perhaps a function name that suggests this opens a dialog?

Reply:
Typically everything that opens a dialog is prefixed with `start`. 
Changed functions to `editDashboardId -> startChangeDashboardId -> execDashboardIdChange`.

Reply:
Better :)